src/block/tile/MonsterSpawner.php

```diff
@@ -130,4 +140,126 @@ protected function addAdditionalSpawnData(CompoundTag $nbt) : void{
 
 		$nbt->setFloat(self::TAG_ENTITY_SCALE, $this->displayEntityScale);
 	}
+
+	public function onUpdate() : bool{
+		if($this->closed || $this->entityTypeId === ":"){
```

Comment:
Why `$this->entityTypeId === ":"` ?

Reply:
A spawner that has no entity to spawn have no need to be ticked